akka-docs/src/main/paradox/typed/actor-discovery.md

```diff
@@ -116,3 +116,9 @@ registered actors that are reachable. The full set of actors, including unreacha
 
 One important difference from local only receptions are the serialization concerns, all messages sent to and back from 
 an actor on another node must be serializable, see @ref:[serialization](../serialization.md).
+
+## Receptionist Scalability
+
+The receptionist does not scale up to any number of services or any number of subscribers or very high turnaround of subscribers and services. 
+It will likely handle up to thousands or tens of thousands of services and subscribers. Use cases with higher 
+demands the receptionist for initial contact between actors on the nodes while the actual logic of those is up to the applications own actors. 
```

Comment:
I would guess that number of subscribers isn't a problem we need to mention here? Or is there some obvious bottleneck for those? They are local only.

Reply:
They are kept in two immutable collections, if you started having millions perhaps. But fair enough, I'll remove that.